contracts/LivepeerProtocol.sol

```diff
@@ -683,9 +680,11 @@ contract LivepeerProtocol {
      * for its time window
      * @param _transcoder Address of transcoder
      */
-    function validRewardTimeWindow(address _transcoder) internal constant returns (bool) {
+    function validRewardTimeWindow(address _transcoder) constant returns (bool) {
```

Comment:
This function is used by the client to poll for when it can call `reward()` so it is now a public function